### PR TITLE
Adding GitHub icon with link to repositorum on footer

### PR DESCRIPTION
**Pull request by Oliweirdo**
## Description

Icon with a link to the repository.

## Motivation and context

#265

## How has this been tested?
 
When click on icon,  new browser tab is open with DAC Repo.

## Screenshots (if appropriate)
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/_Layout.cshtml [hunk 1]
```diff
@@ -123,6 +123,7 @@
                             <li><a href="https://www.facebook.com/devadventcalendar/" target="_blank"><span class="icofont icofont-social-facebook"></span></a></li>
                             <li><a href="https://twitter.com/dev_advent_cal" target="_blank"><span class="icofont icofont-social-twitter"></span></a></li>
                             <li><a href="https://www.youtube.com/channel/UCVyWg3P3hqUfE70btpVobfQ" target="_blank"><span class="icofont icofont-social-youtube-play"></span></a></li>
+                            <li><a href="https://github.com/DevAdventCalendar" target="_blank"><span class="icofont icofont-social-github"></span></a></li>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/_Layout.cshtml]: please use https://github.com/DevAdventCalendar/DevAdventCalendar link

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -126,6 +126,7 @@
                             <li><a href="https://www.facebook.com/devadventcalendar/" target="_blank"><span class="icofont icofont-social-facebook"></span></a></li>
                             <li><a href="https://twitter.com/dev_advent_cal" target="_blank"><span class="icofont icofont-social-twitter"></span></a></li>
                             <li><a href="https://www.youtube.com/channel/UCVyWg3P3hqUfE70btpVobfQ" target="_blank"><span class="icofont icofont-social-youtube-play"></span></a></li>
+                            <li><a href="https://github.com/DevAdventCalendar" target="_blank"><span class="icofont icofont-social-github"></span></a></li>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: please use https://github.com/DevAdventCalendar/DevAdventCalendar link

File: README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
 
-|Build  |Deployment
-|:-----:|:--------:|
-|[![Build](https://github.com/DevAdventCalendar/DevAdventCalendar/workflows/Docker%20Image%20CI/badge.svg)](https://github.com/DevAdventCalendar/DevAdventCalendar/actions?query=workflow%3A%22Docker+Image+CI%22)|[![Deploy](https://vsrm.dev.azure.com/plotzwi/_apis/public/Release/badge/e2ad85fa-38da-4937-a85f-997b254f4cda/1/1)](https://dev.azure.com/plotzwi/DevAdventCalendar/_release?_a=releases&view=mine&definitionId=1)
+|Build  |Deployment| Quality |
+|:-----:|:--------:|:-------:|
+|[![Build](https://github.com/DevAdventCalendar/DevAdventCalendar/workflows/Docker%20Image%20CI/badge.svg)](https://github.com/DevAdventCalendar/DevAdventCalendar/actions?query=workflow%3A%22Docker+Image+CI%22)|[![Deploy](https://vsrm.dev.azure.com/plotzwi/_apis/public/Release/badge/e2ad85fa-38da-4937-a85f-997b254f4cda/1/1)](https://dev.azure.com/plotzwi/DevAdventCalendar/_release?_a=releases&view=mine&definitionId=1)|[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=DevAdventCalendar_DevAdventCalendar&metric=alert_status)](https://sonarcloud.io/dashboard?id=DevAdventCalendar_DevAdventCalendar)|
```
Comment (WTobor) [inline README.md]: these changes are out of scope

Comment (Oliweirdo) [reply README.md]: I don`t know what happened here

Comment (WTobor) [reply README.md]: so please revert these changes :)

Comment (Oliweirdo) [reply README.md]: ok done
